packages/core/src/openapi/openapi-request-builder.spec.ts

```diff
@@ -68,16 +91,67 @@ describe('openapi-request-builder', () => {
           limit: 100
         }
       },
-      { fetchCsrfToken: true }
+      { fetchCsrfToken: true },
+      undefined
+    );
+  });
+
+  it('executes a request using the (iss) token instead of the whole jwt', async () => {
```

Comment:
```suggestion
  it('executes a request using the (iss) token instead of the whole JWT', async () => {
```

Reply:
I am missing the fact that `iss` is passed through options. To me this sounds like we are using only `iss` from the JWT, not the whole token.
Also, isn't `iss` a URL, not a token?

Reply:
On line 126 it is passed. The reason for the difference of passing JWT or the iss (and creating a JWT from it) is, that we do a validation if we get the `userJwt` property and not validation if we have the `iss`.

Reply:
I adjusted also the test name